src/v/cloud_topics/level_one/common/file_io.cc

```diff
@@ -40,18 +41,28 @@ class staging_file_impl : public staging_file {
           _path.native(),
           ss::open_flags::rw | ss::open_flags::truncate
             | ss::open_flags::create);
+        ss::file_output_stream_options options{};
+        // The read buffer size also makes sense as the write buffer here
+        // (default 128KiB).
+        options.buffer_size
+          = config::shard_local_cfg().storage_read_buffer_size();
+        // Defaults to 1, which is reasonable for write-behind as well.
+        options.write_behind
+          = config::shard_local_cfg().storage_read_readahead_count();
```

Comment:
@dotnwat do we have flags for this? For now it's probably OK, but we might want to use different configs for write settings